orchagent/pfc_detect_barefoot.lua

```diff
@@ -21,70 +21,81 @@ for i = n, 1, -1 do
     local is_deadlock = false
     local pfc_wd_status = redis.call('HGET', counters_table_name .. ':' .. KEYS[i], 'PFC_WD_STATUS')
     local pfc_wd_action = redis.call('HGET', counters_table_name .. ':' .. KEYS[i], 'PFC_WD_ACTION')
-    if pfc_wd_status == 'operational' or pfc_wd_action == 'alert' then
```

Comment:
Is this file identical to Mellanox too? If yes, will the symlink work?

Reply:
Since it was there before and it's related to a different platform It's better to leave it as is.